src/components/ImportDropdown/ImportDropdown.jsx

```diff
@@ -21,9 +21,7 @@ const ImportDropdown = ({ label, icon }) => {
   const client = useClient()
   const [showModal, setShowModal] = useState(false)
 
-  const { setIsDialogModalOpen, setDialogModalLabel } = useContext(
-    DialogModalContext
-  )
+  const { setIsDialogModalOpen } = useContext(DialogModalContext)
```

Comment:
If a context provider is not set by upper components, then you'll get a deconstruction error like `cannot read property 'setIsDialogModalOpen' of undefined`.

When using `useContext(...)` I like to throw an understandable message if undefined before using it:
```js
const dialogModalContext = useContext(DialogModalContext)
if (!dialogModalContext) {
 throw new Error('ImportDropdown should be in DialogModalProvider')
}
const { setIsDialogModalOpen } = dialogModalContext
```
It's a bit more verbose but this would ease future debugging sessions if component is reused.

Reply:
Bien vu, en effet ta proposition est plus viable, je vais ajouter 2 hooks types `useDialogModalContext` & `useStepperContext` pour gérer ça 👍